piff/model.py

```diff
@@ -154,29 +154,28 @@ def write(self, fits, extname):
         data = numpy.array(zip(*cols), dtype=dtypes)
         fits.write_table(data, extname=extname)
 
-    @classmethod
-    def readKwargs(cls, fits, extname):
-        """Read the kwargs from the data in a FITS binary table.
+        # now write the parameters
+        self.writeParameters(fits, extname + '_parameters')
 
-        The base class implementation just reads each value in the table and uses the column
-        name for the name of the kwarg.  However, derived classes may want to do something more
-        sophisticated.  Also, they may want to read other extensions from the fits file
-        besides just extname.
+    def writeParameters(self, fits, extname):
+        """Write parameters of Model to a FITS file.
 
-        :param fits:        An open fitsio.FITS object.
-        :param extname:     The name of the extension with the model information.
+        By default does nothing.
 
-        :returns: a kwargs dict to use to initialize the model
+        :param fits:        An open fitsio.FITS object
+        :param extname:     The name of the extension to write the model information.
         """
-        cols = fits[extname].get_colnames()
-        # Remove 'type'
-        assert 'type' in cols
-        cols = [ col for col in cols if col != 'type' ]
+        pass
 
-        data = fits[extname].read()
-        assert len(data) == 1
-        kwargs = dict([ (col, data[col][0]) for col in cols ])
-        return kwargs
+    def readParameters(self, fits, extname):
+        """Read parameters of Model from a FITS file.
+
+        By default does nothing.
+
+        :param fits:        An open fitsio.FITS object
+        :param extname:     The name of the extension to write the model information.
+        """
+        pass
```

Comment:
Is the protocol here (since two different branches simultaneously came up with this functionality) for me to just rename the function and then when #20 gets merged in let it be overwritten?